module-1/src/Components/Counter/Counter.tsx

```diff
@@ -1,26 +1,43 @@
-import React, { useState } from 'react'
+import { useState } from "react";
 
 interface ICounterProps {
-    initialValue?: number;
-  }
+  /**
+   * This is the initial value of the Counter
+   */
+  initialValue?: number;
+}
+
+export const Counter = ({ initialValue = 0 }: ICounterProps) => {
+  const [count, setCount] = useState(initialValue);
+  const increment = () => {
+    setCount(count + 1);
+  };
 
-const Counter:React.FC<ICounterProps> = ({initialValue=0}) => {
-    const [count, setCount] = useState(initialValue);
-    const increment = () => {
-        setCount(count + 1);
-      };
-    
-    const decrement = () => {
-        setCount(count - 1);
-      };
+  const decrement = () => {
+    setCount(count - 1);
+  };
   return (
-    <div className='border p-3 m-3'>
-      <h3 data-testid="Cy-Counter-Title" className='text-lg m-4 font-bold'>Counter Component</h3>
-       <p data-testid="Cy-Counter-Text" className='text-3xl m-2'>Counter: {count}</p>
-       <button data-testid="Cy-Counter-Increment" className='ml-2 bg-blue-500 hover:bg-blue-700 text-white py-1 px-2 border border-blue-700 rounded' onClick={increment}>Increment</button>
-       <button data-testid="Cy-Counter-Decrement" className='ml-2 bg-blue-500 hover:bg-blue-700 text-white py-1 px-2 border border-blue-700 rounded' onClick={decrement}>Decrement</button>
+    <div className="border p-3 m-3">
+      <h3 data-testid="Cy-Counter-Title" className="text-lg m-4 font-bold">
+        Counter Component
+      </h3>
+      <p data-testid="Cy-Counter-Text" className="text-3xl m-2">
+        Counter: {count}
+      </p>
+      <button
+        data-testid="Cy-Counter-Increment"
```

Comment:
By default all buttons have `type="submit"`. I recommend adding `type="button"` to prevent accidental weird behaviors. This is applicable not just this file but throughout the project.

Reply:
fixed